Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/ExternalActionHelper.kt

```diff
@@ -71,12 +72,18 @@ object ExternalActionHelper {
      */
     fun openUrl(fragment: Fragment, url: String) {
         try {
+            val customTabBuilder = CustomTabsIntent.Builder().build()
+            fragment.context?.let { customTabBuilder.launchUrl(it, Uri.parse(url)) }
```

Comment:
This still launches the chooser dialog if no default browser is set.

Also, better to use fragment.requireContext() for better readability, IMHO.

Reply:
I have changed it to use requireContext()

What about the chooser dialog?